source/streaming-mode/streaming-read-config.txt

```diff
@@ -46,6 +46,26 @@ You can configure the following properties when reading data from MongoDB in str
    * - ``collection``
      - | **Required.**
        | The collection name configuration.
+       | You can specify multiple collections by separating each collection name
+         with a comma (,). Do not add a space between the collections
+         unless the space is a part of the collection name.
+       | Specify multiple collections as follows:
+
+       .. code-block:: java
+
+          ...
+          .option("spark.mongodb.collection", "collectionOne,collectionTwo")
+
+       | You can scan from all collections in the database by using an asterisk (*).
+       | Specify all collections as follows:
```

Comment:
```suggestion
       | You can scan from all collections in the database by passing an asterisk (*) as a string for the collection name.
       | Specify all collections as follows:
```

feel free to play with this part; i think it's worth it to specify that it needs to be in quotes (though the compiler should catch this)